src/content/documentation/develop/best-practices-for-collecting-user-data-consents.md

```diff
@@ -148,16 +148,14 @@ Before designing your consent dialogs, you should determine the privacy consent
 In this example:
 
 1. Your extension should start up with all data collection functionality disabled.
-2. Shortly after installation (see [Best practices for onboarding, upboarding, and offboarding users](/documentation/develop/onboard-upboard-offboard-users/)), present them with a consent dialog.
+2. Immediately after installation (see [Best practices for onboarding, upboarding, and offboarding users](/documentation/develop/onboard-upboard-offboard-users/)), present them with a consent dialog.
    On this dialog:
 3. The option for collecting personally identifying information must default to disabled so that the user has to actively opt-in.
 4. The options for technical data may default to enabled, so that the user may choose to opt-out.
 5. If the user:
-    - opts IN to all features &rarr; continue with all extension features enabled.
-    - opts OUT of any feature that cannot be disabled &rarr; offer them the option to remove the extension. If they choose not to remove the extension present the consent dialog again.
-    - opts OUT only from features that can be disabled &rarr; disable those features and continue running the extension.
-
-You can prompt the user to uninstall the extension with [`management.uninstallSelf()`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/management/uninstallSelf).
+    - accepts mandatory and optional data collection &rarr; continue with data collection enabled enabled.
+    - declines mandatory data collection &rarr; uninstall the extension with [`management.uninstallSelf()`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/management/uninstallSelf).
+    - declines optional data collection &rarr; disable optional data collection and continue running the extension.
```

Comment:
Swap the order of these bullets so the sequence of choices appears in order of desirability for the developer.
```suggestion
    - declines optional data collection &rarr; disable optional data collection and continue running the extension.
    - declines mandatory data collection &rarr; uninstall the extension with [`management.uninstallSelf()`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/management/uninstallSelf).
```